lib/CodeGen/CodeGenModule.cpp

```diff
@@ -705,6 +705,30 @@ void CodeGenModule::initEEIDeclaration() {
   Func_getAddress->addFnAttr(Ethereum);
   Func_getAddress->addFnAttr(
       llvm::Attribute::get(VMContext, "wasm-import-name", "getAddress"));
+
+  // getCodeSize
+  FT = llvm::FunctionType::get(Int32Ty, {}, false);
+  Func_getCodeSize = llvm::Function::Create(FT, llvm::Function::ExternalLinkage,
+                                           "ethereum.getCodeSize", TheModule);
+  Func_getCodeSize->addFnAttr(Ethereum);
+  Func_getCodeSize->addFnAttr(
+      llvm::Attribute::get(VMContext, "wasm-import-name", "getCodeSize"));
+
+  // getExternalCodeSize
+  FT = llvm::FunctionType::get(Int32Ty, {AddressPtrTy}, false);
```

Comment:
Use `Int32PtrTy` instead of `AddressPtrTy`.

`AddressPtrTy` is for EVM backend which represents 160 bits integer.